src/frontend/interfaces/action/build-action-click-handler.ts

```diff
@@ -32,7 +35,7 @@ export const buildActionClickHandler = (
       params, action, actionResponseHandler,
     })
 
-    if (action.guard && !confirm(action.guard)) {
+    if (action.guard && !confirm(translateMessage(action.guard))) {
```

Comment:
this should use `resource.id` as second param I think

Reply:
added